test/TestProjects/MgmtKeyvault/tests/Generated/Mock/DeletedManagedHsmCollectionTest.cs

```diff
@@ -24,7 +23,7 @@ public DeletedManagedHsmCollectionMockTests(bool isAsync) : base(isAsync, Record
         }
 
         [RecordedTest]
-        public async Task Get()
+        public async System.Threading.Tasks.Task Get()
```

Comment:
This change will go away if you change the return type of `GetTaskOrVoid` from string to `FormattableString`